src/provider/zeebe/properties/FormProps.js

```diff
@@ -35,55 +41,172 @@ export function FormProps(props) {
     return [];
   }
 
-  return [
+  const entries = [];
+
+  entries.push(
+    {
+      id: 'formType',
+      component: FormType,
+      isEdited: isSelectEntryEdited
+    },
```

Comment:
I did this because I got an error when trying to do ` injector.invoke(FormHelper)` in the main component FormProps: `unhandled error in event listener TypeError: Cannot read properties of null (reading 'invoke')`. Do you know why this might happen?

Reply:
`injector` is not defined, that is what the error says.

Reply:
But I'm trying to do:
```
const injector = useService('injector');
const formHelper = injector.invoke(FormHelper);
```
Like I do in the other components, and only on the main one I get that error.

Reply:
Could you check-in a failing test case? I'm happy to have a look.

Reply:
I added a test branch with these changes [here](https://github.com/bpmn-io/bpmn-js-properties-panel/tree/WIP-custom-form-key). More specifically:
https://github.com/bpmn-io/bpmn-js-properties-panel/blob/84b067a02b3619fda9cf4ee1730875caca235192/src/provider/zeebe/properties/FormProps.js#L41

This causes a bunch of tests to fail because it throws the error I mentioned.